### PR TITLE
Merge time tab

### PR DESCRIPTION
**Pull request by MajorBreakfast**

### CODE REVIEW

File: ionic2/app/components/filter-popover/filter-popover.component.ts [hunk 1]
```diff
@@ -1,33 +1,20 @@
-import { Component } from '@angular/core';
-import { FilterPopoverTabTime } from '../filter-popover-tab-time/filter-popover-tab-time';
-import { FilterPopoverTabPokemon } from '../filter-popover-tab-pokemon/filter-popover-tab-pokemon';
-import { Events, ViewController } from 'ionic-angular';
+import { Component } from "@angular/core";
```
Comment (philbu) [inline ionic2/app/components/filter-popover/filter-popover.component.ts]: " -> '

Comment (AlexanderLill) [inline ionic2/app/components/filter-popover/filter-popover.component.ts]: Please use single quotation marks :)

File: ionic2/app/components/poke-filter-time-tab/poke-filter-time-tab.component.html [hunk 2]
```diff
@@ -0,0 +1,56 @@
+
```
Comment (AlexanderLill) [inline ionic2/app/components/poke-filter-time-tab/poke-filter-time-tab.component.html:1]: Please remove empty starting line

File: ionic2/app/components/poke-filter-time-tab/poke-filter-time-tab.component.ts [hunk 3]
```diff
@@ -0,0 +1,46 @@
+import { Component, ElementRef, OnInit, ViewChild } from '@angular/core';
+import { Tab,  } from 'ionic-angular';
```
Comment (AlexanderLill) [inline ionic2/app/components/poke-filter-time-tab/poke-filter-time-tab.component.ts:2]: Please remove comma

File: ionic2/app/components/poke-filter-time-tab/poke-filter-time-tab.component.ts [hunk 4]
```diff
@@ -0,0 +1,46 @@
...
+  onSightingsToggleChange(toggle) {
+    this.sightingsRangeValue = toggle.checked ? 4 : 9; // '1d' or 'off'
+  }
+
+  onPredictionsRangeChange() {
+    this.arePredictionsShown = this.predictionsRangeValue !== 0;
+  }
+
+  onPredictionsToggleChange(toggle) {
+    console.log('jojojojoj')
```
Comment (AlexanderLill) [inline ionic2/app/components/poke-filter-time-tab/poke-filter-time-tab.component.ts]: Do we still need that? :)

**Review by AlexanderLill — CHANGES_REQUESTED** [review]

**Review by AlexanderLill — APPROVED** [review]

**Pull request merged**